index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+   <div class="container">
+       <header>
+           <nav class="navItems">
+            <h1>Welcome</h1>
+                    <i class="fa-solid fa-bars"></i>
+           </nav>
+       </header>
+    <section class="headline">
+           <h1>Hey there.I'm <br> Precious <br>
+               <span> I'm a software developer</span>
+           </h1>
+           <h3 id="headline_text"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            do not hestiate to contact me.
+           </h3>
+
+           <section class="socials">
+               <h3>
+               <a href="#">
+                   <img src="icons/twitter.png" alt="twitter_link">
+                </a>
+               <a href="#">
+                   <img src="icons/Linkedin.png" alt="linkedin_link">
+                </a>
+                <a href="#">
+                    <img src="icons/medium.png" alt="medium_link">
+                 </a>
+               <a href="#">
+                   <img src="icons/github.png" alt="github_link">
+                </a>
+               <a href="#">
+                   <img src="icons/angellist.png" alt="angellist_link">
```

Comment:
- [ ] Let's change something here too, from that `h3` tag, please change it to a `ul` tag and wrap each an inside a `li` tag then 😄 I think it makes sense doing that
- [ ] Please use proper link to each of your social medial account